src/engine/world/instances.ts

```diff
@@ -97,7 +97,7 @@ export class WorldInstance {
      * @param instanceId [optional] The instanceId to apply to this new world instance.
      * If not provided, the instance will be considered a public global world instance.
      */
-    public constructor(public readonly instanceId: string = null) {
+    public constructor(public readonly instanceId: string) {
```

Comment:
This should be optional according to the docs

Reply:
doesn't need to be, and makes the API more complicated, so have removed